### ISSUE TITLE
[Request] Left Pane Sorting

### ISSUE THREAD

**Issue by Astor1977** [TanninOne/modorganizer#941]
I really enjoyed MO sorting my left pane in Skyrim, but MO 207 doesn't do that.  This is making it more difficult to pin down my random ctd when working at a crafting bench as i'm unable to figure out what mod should go where.  

Is this feature on the "to do list"?

**Comment by Gernash** [comment]
run fo4edit it will give errors and stop showing you load priority if loot does not work it out for you

**Comment by Astor1977** [comment]
LOOT is unreliable at best.  It doesn't seem to sort a lot of mods, or load order for things like armor mods and such just don't matter anymore.

As for FO4Edit, i'm running an error check right now, but won't know what to do with any of it after it's done. I just hit Ctrl+A then error check; it's taking a while.

**Comment by Gernash** [comment]
Ah... well if you have a mod loder issue then running fo4edit will tellyou if priority is wrong if you have issues with eg.Valdacils Item Sorting and Functional displays.  then you have to look at the mod overwrite order.....

bacically load all your mods and right click left folder pick apply filter then apply
after a while you will see all the mods red is conflict orange is not that bad green is ok 
basicall if you have an issue with a particulat mod you drill down to the red and in the right side you work out what is overwriting it then change the order ot make an overwrite mod.

There is heaps of youtube on the topic for skyrim just look around.

**Comment by Gernash** [comment]
there will always be overwrites the choice it yours what you want to be overwritten

**Comment by Astor1977** [comment]
Yeah... I can't even begin to understand FO4EDIT.  It's just a mass of red and green boxes with tons of crap that means absolutely nothing to me.  No clue where to even find what's overriding what.  My brain literally melted into a steaming pile of goo when i tried to sort that s*** out.

**Comment by Gernash** [comment]
You would have to cite examples of that is causing CDT's all the Fallout 4 mods are quite primative (not including CBBE/bodyslide or any other external exe driven mods I have not seen any)

HAHAHHAHHA all good I'll find you a link 

IT IS REALLY BASIC I assure you

**Comment by Astor1977** [comment]
So far it's playing fine, except at the crafting benches which leads me to think of a conflict between Armorsmith Keywords, Concealed Armors, Crafting Workbenches, and possible Weaponsmith, as to my knowledge those are the only mods i have installed that affect workbenches.

**Comment by Gernash** [comment]
https://www.youtube.com/watch?v=CQi7vnqXySc
https://www.youtube.com/watch?v=gKop6O2z9J0
https://www.youtube.com/watch?v=-h-SkZK-Tts

**Comment by Astor1977** [comment]
Thank you. checking those out now

**Comment by Gernash** [comment]
Basically most overwrites just overwrite weapon damage and item sorting mods. CDT are mainly caused by House mods.
otherwise you have fiddled with your ini too much or made an error with enb setup

**Comment by Astor1977** [comment]
my rig cannot handle ENB, so i don't have one installed.  I do have homemaker and ski/ske installed.
